app/main.py

```diff
@@ -1,13 +1,19 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return ({"line" if key == "line_number"
+            else "column" if key == "column_number"
+            else "message" if key == "text"
+            else "name" if key == "code" else key: value
+            for key, value in error.items()
+            if key not in ("filename", "physical_line")}
+            | {"source": "flake8"})
```

Comment:
Remove the `if else` conditions and the dict comprehension. Hardcode the structure of the dict that you return instead for better readability. There is no need to place it inside of `()`, return it directly.